### DISCUSSION TITLE
Switch to using main branch

### DISCUSSION THREAD

**Discussion by RebeccaStevens** [jonaskello/eslint-plugin-functional#312] [Ideas]
@jonaskello Should we rename `master` to `main` as that is the new standard?

**Comment by jonaskello** [comment]
Yes, the name change is totally OK with me so if you'd like you can go ahead and change it.

**Reply by RebeccaStevens** [reply]
Alright; I'll setup a PR to change over all changes to the CI and that are are needed. Once done; you'll need to modify the repo settings to change what the default branch is.
